syntax = "proto3";

message Replyresult{
	bytes image = 1;
	int32 track_id = 2;
	int32 msg = 3;
}

message RequestFace{
	bytes images = 1;
	string action = 2;
	string name = 3;
	bytes track_ids = 4;
	int32 frame_index = 5;
}


// responce server
service DataProcessing{
	rpc getStream (RequestFace) returns (Replyresult) {}
}